system_tests/espresso/chain/block_builder.go

```diff
@@ -0,0 +1,155 @@
+package chain
```

Comment:
nit: can we move the code related to testing load of Molten in another PR? It doesnt seem related to the refactoring transaction streamer. Smaller PRs help with better reviews 🙏

Reply:
Sure